src/utils/commands.ts

```diff
@@ -47,6 +47,15 @@ export abstract class AdminCommand extends BaseCommand {
   }
 }
 
+export abstract class MentorCommand extends BaseCommand {
```

Comment:
This seems specific to the bootcamp event, maybe move it to a bootcamp specific file instead? Also could you add some comments explaining who's considered a mentor and how that correlates to the perms required to use the command?